WowUp.WPF/Utilities/FileUtilities.cs

```diff
@@ -168,5 +168,12 @@ public static void DirectoryCopy(string sourceDirName, string destDirName, bool
                 }
             }
         }
+
+        public static void OpenInFileManager(string path)
```

Comment:
This function seems fine, but can we use the same format as StringExtensions.OpenUrlInBrowser which does basically the same thing.